### DISCUSSION TITLE
Issues with homebank install

### DISCUSSION THREAD

**Discussion by davidmickelsen** [Homebrew/discussions#5213] [Casks]
### Output of `brew config`

```shell
HOMEBREW_VERSION: 4.2.13
ORIGIN: https://github.com/Homebrew/brew
HEAD: bb753a6c48575ae018d7ad925d7762d59647fff7
Last commit: 3 days ago
Core tap JSON: 19 Mar 17:05 UTC
Core cask tap JSON: 19 Mar 17:05 UTC
HOMEBREW_PREFIX: /usr/local
HOMEBREW_CASK_OPTS: []
HOMEBREW_MAKE_JOBS: 4
Homebrew Ruby: 3.1.4 => /usr/local/Homebrew/Library/Homebrew/vendor/portable-ruby/3.1.4/bin/ruby
CPU: quad-core 64-bit skylake
Clang: 14.0.0 build 1400
Git: 2.44.0 => /usr/local/bin/git
Curl: 8.4.0 => /usr/bin/curl
macOS: 12.7.3-x86_64
CLT: 14.2.0.0.1.1668646533
Xcode: N/A
```

### Output of `brew doctor`

```shell
Please note that these warnings are just used to help the Homebrew maintainers
with debugging if you file an issue. If everything you use Homebrew for is
working fine: please don't worry or file an issue; just ignore this. Thanks!

Warning: Some installed kegs have no formulae!
This means they were either deleted or installed manually.
You should find replacements for the following formulae:
  python-mutagen
  python-brotli
```

### Description of issue

I installed homebank using `brew install homebank`. I was expecting a cask, since HomeBank is a GUI program; however, it seems to have installed a formula. When I run `brew ls`, `homebank` appears under Formulae, but not under Casks. Anyway, when I subsequently run `homebank`, I get the following message:

`** (homebank:13932): WARNING **: 15:18:39.266: unable to load file /Users/davidmickelsen/.config/homebank/preferences: No such file or directory`

What do I need to do to get HomeBank up and running?

### Relevant casks

homebank

**Comment by daeho-ro** [comment]
```
brew search homebank
==> Formulae
homebank                                                  homeshick
```

**Comment by SMillerDev** [comment]
> What do I need to do to get HomeBank up and running?

That really seems more like a question for the homebank devs
